packages/@sanity/form-builder/src/inputs/PortableText/PortableTextInput.tsx

```diff
@@ -84,12 +120,16 @@ const PortableTextInputWithRef = React.forwardRef(function PortableTextInput(
 
   const toast = useToast()
 
-  const editorId = useMemo(() => uniqueId('PortableTextInputRoot'), [])
   // Memoized patch stream
   const patches$: Subject<EditorPatch> = useMemo(() => new Subject(), [])
   const patchObservable = useMemo(() => patches$.asObservable(), [patches$])
 
-  const handleToggleFullscreen = useCallback(() => setIsFullscreen(!isFullscreen), [isFullscreen])
+  const innerElementRef = useRef(null)
+
+  const handleToggleFullscreen = useCallback(() => {
+    setIsFullscreen(!isFullscreen)
```

Comment:
Small thing - I prefer this approach for toggle boolean in a state: `setIsFullscreen((v) => !v)`

This might also be more performant, because `isFullscreen` can be removed from the `useCallback` dependency array, and the function will not be re-created when that value changes.